packages/js/src/browser/integrations/breadcrumbs.ts

```diff
@@ -1,267 +1,319 @@
-              const message = inspectArray(args)
+              const args = Array.prototype.slice.call(arguments);
+              const message = inspectArray(args);
               const opts = {
                 category: 'log',
                 metadata: {
                   level: level,
-                  arguments: sanitize(args, 3)
-                }
-              }
+                  arguments: sanitize(args, 3),
+                },
+              };
 
-              client.addBreadcrumb(message, opts)
+              client.addBreadcrumb(message, opts);
 
               if (typeof original === 'function') {
-                Function.prototype.apply.call(original, _window.console, arguments)
+                Function.prototype.apply.call(
+                  original,
+                  _window.console,
+                  arguments
+                );
               }
-            }
-          })
-        })
+            };
+          });
+        });
       })();
 
       // Breadcrumbs: instrument click events
       (function () {
-        if (!breadcrumbsEnabled('dom')) { return }
-
-        _window.addEventListener('click', (event) => {
-          let message, selector, text
-          try {
-            message = stringNameOfElement(event.target as HTMLElement)
-            selector = stringSelectorOfElement(event.target)
-            text = stringTextOfElement(event.target)
-          } catch (e) {
-            message = 'UI Click'
-            selector = '[unknown]'
-            text = '[unknown]'
-          }
+        if (!breadcrumbsEnabled('dom')) {
+          return;
+        }
 
-          // There's nothing to display
-          if (message.length === 0) { return}
+        _window.addEventListener(
+          'click',
+          (event) => {
+            let message, selector, text;
+            try {
+              message = stringNameOfElement(event.target as HTMLElement);
+              selector = stringSelectorOfElement(event.target);
+              text = stringTextOfElement(event.target);
+            } catch (e) {
+              message = 'UI Click';
+              selector = '[unknown]';
+              text = '[unknown]';
+            }
 
-          client.addBreadcrumb(message, {
-            category: 'ui.click',
-            metadata: {
-              selector,
-              text,
-              event
+            // There's nothing to display
+            if (message.length === 0) {
+              return;
             }
-          })
-        }, true)
+
+            client.addBreadcrumb(message, {
+              category: 'ui.click',
+              metadata: {
+                selector,
+                text,
+                event,
+              },
+            });
+          },
+          // In CloudFlare workers this must be false. This is a hacky way to detect it.
+          // CF workers require useCapture to be false.
+          _window.location ? true : false
```

Comment:
Shouldn't you `return` here?

Reply:
Shouldnt need one, this is a parameter being passed to the event listener.

Reply:
Oh, ignore what I said. I thought this was the return of a function 🤦